### PR TITLE
add fuzzy-rough-learn

### PR DESCRIPTION
**Pull request by oulenz**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/fuzzy-rough-learn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ac850faf3397c531c80bca0b7ade0e0525a8f04714a915d4d54be519b44905fe
+
+build:
+  number: 0
+  skip: True  # [py<37]
```
Comment (marcelotrevisani) [inline recipes/fuzzy-rough-learn/meta.yaml]: Please remove the ``skip`` here, and I think this recipe could be ``noarch: python`` right?
With ``noarch`` you can just select which Python version you want to use
```suggestion
  noarch: python
```

File: recipes/fuzzy-rough-learn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/fuzzy-rough-learn/meta.yaml]: Please also select it for Python ``3.7``
```suggestion
    - python >=3.7.0
```

**Comment by oulenz** [comment]
Thanks for the review! I've fixed the two issues.

- I hadn't put in `noarch: python` with an eye to the future because I expect to add cython code, but I guess I can change it at that point.
- while the instructions are mostly very easy to follow, I guess they could be clearer re how exactly to specify the python version
- specifically, could you explain why `skip: True  # [py<37]` is wrong here?

**Comment by marcelotrevisani** [comment]
> Thanks for the review! I've fixed the two issues.
> 
>     * I hadn't put in `noarch: python` with an eye to the future because I expect to add cython code, but I guess I can change it at that point.
> 
>     * while the instructions are mostly very easy to follow, I guess they could be clearer re how exactly to specify the python version
> 
>     * specifically, could you explain why `skip: True  # [py<37]` is wrong here?

The ``skip`` is not wrong, but if this recipe can be ``noarch`` we cannot use selectors on ``noarch`` recipes. That is why I suggested adding the ``>= 3.7.0``
Sorry if I was not clear

**Pull request merged**